README.md

```diff
@@ -69,6 +69,13 @@ fast_paths::save_to_disk(&fast_graph, "fast_graph.fp");
 let fast_graph = fast_paths::load_from_disk("fast_graph.fp");
 ```
 
+To be able to use the graph in a 32bit WebAssembly environment it needs to be transformed to a 32bit representation when preparing it on a 64bit system. This can be achieved with the following two methods, but it will only work for graphs that do not exceed the 32bit limit.
```

Comment:
"for graphs that do not use more than 2^32 edges, nodes, or weight values"
Maybe this is a little more clear what the "32bit limit" means